index.html

```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
              <div class="ellipse8"></div>
              <div class="ellipse15"></div>
          </div>
+         <div class="flex-botu">
+            <div class="resume">Get my resume</div>
```

Comment:
- [ ] As requested by the previous code reviewer, this is a button, please kindly fix that.